### PR TITLE
Finish clarifying pointer arguments

### PR DESCRIPTION
**Pull request by kgaillot**
This wraps up the project to clarify pointer arguments in functions that have doxygen blocks. This one handles tools.

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , here's the last one of these!

**Review by nrwahl2 — COMMENTED** [review]
Woohoo!

File: tools/crm_resource_runtime.c [hunk 1]
```diff
@@ -268,18 +271,21 @@ cli_resource_update_attribute(pe_resource_t *rsc, const char *requested_name,
     const char *common_attr_id = attr_id;
 
     if (attr_id == NULL && force == FALSE) {
-        find_resource_attr (out, cib, XML_ATTR_ID, uber_parent(rsc)->id, NULL,
+        find_resource_attr (out, cib, XML_ATTR_ID,
+                            pe__const_top_resource(rsc, false)->id, NULL,
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply tools/crm_resource_runtime.c:275]: Almost certainly this should be going up to the bundle, but that would be its own project

File: tools/crm_resource_runtime.c [hunk 2]
```diff
@@ -1120,8 +1120,8 @@ static void display_list(pcmk__output_t *out, GList *items, const char *tag)
  *
  * This also updates the working set timestamp to the current time.
  *
- * \param[in] data_set   Working set instance to update
- * \param[in] xml        XML to use as input
+ * \param[in,out] data_set  Working set instance to update
+ * \param[in]     xml       XML to use as input
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply tools/crm_resource_runtime.c:1124]: Whoops, missed that

**Comment by kgaillot** [comment]
Added a couple commits on top to address review

**Review by nrwahl2 — APPROVED** [review]
Sure feels good to be done with these. I'm sure you agree.

**Pull request merged**